algorithms/symmetry/cosym/target.py

```diff
@@ -86,7 +86,9 @@ def __init__(
 
         self._sym_ops = OrderedSet(["x,y,z"])
         self._lattice_group = lattice_group
-        self._sym_ops.update({op.as_xyz() for op in self._generate_twin_operators()})
+        self._sym_ops.update(
+            OrderedSet(op.as_xyz() for op in self._generate_twin_operators())
+        )
```

Comment:
```suggestion
        self._sym_ops.update(op.as_xyz() for op in self._generate_twin_operators())
```
Possibly?

Reply:
e.g. it was the going via set that was the problem and `OrderedSet.update` only expects that it's input is in the order you want to preserve